docs/getting-started.md

```diff
@@ -199,3 +199,26 @@ Added storage nodestorage to nodejs
 ```
 
 That's it! Storage has been added your component with an allocated size of 1 Gb.
+
+## Extra OpenShift notes
+
+These are some extra installation / getting started instructions for your local OpenShift cluster.
+
+### Service Catalog
+
+In order to use the Service Catalog it must be enabled within your OpenShift cluster.
+
+If you are using `minishift` you'll need to start an OpenShift cluster with a version 3.9.0+ and Service Catalog explicitly enabled.
+
+```sh
+# Install the minishift addon
+git clone https://github.com/minishift/minishift-addons
```

Comment:
we don't need this addon now

Reply:
@surajnarwade why not? is it integrated into the newest minishift now?